packages/hoppscotch-common/src/services/scroll.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Service } from "dioc"
+
+export class ScrollService extends Service {
```

Comment:
we can add some documentation on the use of the service and each classes, also can you add the test case for this service

Reply:
you can add the test file in services/__tests__/scroll.service.spec.ts